src/test/script_standard_tests.cpp

```diff
@@ -598,7 +598,7 @@ BOOST_AUTO_TEST_CASE(script_standard_IsMine)
         BOOST_CHECK(keystore.AddKey(keys[1]));
 
         CScript redeemScript = GetScriptForMultisig(2, {uncompressedPubkey, pubkeys[1]});
-        scriptPubKey = GetScriptForDestination(CScriptID(redeemScript));
+        scriptPubKey = GetScriptForDestination(ScriptHash(CScriptID(redeemScript)));
```

Comment:
`ScriptHash(redeemScript)` again?

Reply:
fixed

Reply:
A followup PR that would make this particular mistake fail the test would be nice though.